src/prefect/orion/schemas/core.py

```diff
@@ -314,11 +318,30 @@ def populate_deprecated_fields(cls, values):
         """
         if not values.get("retries", None) and values.get("max_retries", 0) != 0:
             values["retries"] = values["max_retries"]
+
         if (
             not values.get("retry_delay", None)
             and values.get("retry_delay_seconds", 0) != 0
         ):
             values["retry_delay"] = values["retry_delay_seconds"]
+
+        return values
+
+    @root_validator
```

Comment:
Can you use a `validator` instead of a `root_validator`? Since these validations are for single fields (without interactions) we don't need to use a root validator.